### PR TITLE
Update 2020-02-25-dap-learning-series-justifying-web-analysts.md

### PR DESCRIPTION
**Pull request by ayerkes**
---

**Preview:**: https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/gsa/digitalgov.gov/adding-tags-and-author-1/event/2020/02/19/dap-learning-series-youtube-event-option/

### CODE REVIEW

File: content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-# View this page at https://digital.gov/event/2020/02/dap-learning-series-justifying-web-analysts
+# View this page at https://digital.gov/event/2020/03/dap-learning-series-justifying-web-analysts
```
Comment (jeremyzilar) [inline content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md]: How did this change? Curious if this was just done with the workflow tool, or if this was done by hand.

File: content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md [hunk 2]
```diff
@@ -11,10 +11,19 @@ registration_url: https://www.eventbrite.com/e/93157648033
 captions: https://www.captionedtext.com/client/event.aspx?EventID=4341007&CustomerID=321
 
 # start date
-date: 2020-03-25 14:00:00 -0500
+date: 2020-03-25 15:00:00 -0500
```
Comment (jeremyzilar) [inline content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md]: Was this time also changed intentionally? Curious to know if the workflow tool accidentally changed it on you.

**Comment by ToniBonittoGSA** [comment]
1) the preview link in the top box does not match the file that was edited
2) only the front matter was updated; not the location `/02/` and filename `2020-02-25-dap-learning-series-justifying-web-analysts.md` at the top

Screen Shot 2020-02-12 at 12 02 08 AM

File: content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md (hunk 1 above)
Comment (DanielJPino-ux) [reply content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md]: this was done via the workflow tool - we added speakers and tags to the page as we are diving deep into the meta-tagging process

Comment (jeremyzilar) [reply content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md]: ok!

**Comment by saracope** [comment]
Can someone clarify what is still needed for this PR? Is it waiting on something specific?

**Comment by saracope** [comment]
Reviewed w/@ayerkes. Closing this out.

**Pull request closed without merging**